frc_characterization/logger_analyzer/data_analyzer.py

```diff
@@ -1028,7 +1035,14 @@ def isRotation(units):
     STATE.test.trace_add("write", defineTestResults)
 
     Label(topFrame, text="Units:", width=10).grid(row=0, column=4)
-    unitChoices = {"Feet", "Inches", "Meters", "Radians", "Rotations", "Degrees"}
+    unitChoices = [
+        Units.DEGREES.value,
+        Units.ROTATIONS.value,
+        Units.RADIANS.value,
+        Units.FEET.value,
+        Units.INCHES.value,
+        Units.METERS.value,
+    ]
     unitsMenu = OptionMenu(topFrame, STATE.units, *sorted(unitChoices))
```

Comment:
no need for "unit choices" as the enums are iterable

```suggestion
    unitsMenu = OptionMenu(topFrame, STATE.units, *sorted(Test.__members__.keys()))
```

Reply:
[Done](https://github.com/Piphi5/frc-characterization/blob/Refactor/frc_characterization/logger_analyzer/data_analyzer.py#L1037) (I thought list comprehension would be more readable)

Reply:
Yeah, it probably is.